imports/ui/components/addAudio/wizard.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react';
+import StepWizard from 'react-step-wizard';
+
+import Nav from './nav';
+import Step1 from './Step1';
+import Step2 from './Step2';
+import Step3 from './Step3';
+import Step4 from './Step4';
+import Step5 from './Step5';
+import Stats from './Stats';
+/**
+ * A basic demonstration of how to use the step wizard
+ */
+export default class Wizard extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            form: {},
+        };
+    }
+
+    updateForm = (key, value) => {
+        const { form } = this.state;
+
+        form[key] = value;
```

Comment:
It isn't advised to modify the object directly. React can only listen to new object references, mutating it won't update the reference. 

```
const newForm = { ...this.state.form, [ke
```